src/i18n/locale-en.json

```diff
@@ -46,9 +46,36 @@
     "externalsync.delete.success.msg": "Deleted connector {{name}}",
     "externalsync.delete.remote.warning": "You may have to remove the index manually from {{key}}",
 
+    "jdbc.not.get.sql.msg": "Could not get SQL table configurations",
+    "jdbc.delete.sql.table.warning.msg": "Are you sure you want to delete the SQL table configuration '{{name}}'?",
+    "jdbc.not.delete.sql.msg": "Could not delete SQL table",
+    "jdbc.warning.unsaved.changes": "You have unsaved changes. Are you sure that you want to exit?",
+    "jdbc.refreshing.namespaces.msg": "Refreshing namespaces",
+    "jdbc.not.get.sql.msg2": "Could not get SQL table configuration",
+    "jdbc.required.configuration.name": "SQL configuration name is required",
+    "jdbc.saved.configuration": "SQL table configuration saved",
+    "jdbc.not.saved.configuration": "Could not save SQL table configuration",
+    "jdbc.configuration.updated": "SQL table configuration updated",
+    "jdbc.not.add.known.prefixes": "Error! Could not add known prefixes",
+    "jdbc.suggest.sql.type": "Suggest SQL type",
+    "jdbc.not.suggest.column.type": "Could not suggest column type",
+    "jdbc.warning.column.type.msg": "Are you sure you want to get suggestions for all columns? This action will overwrite all column type mappings!",
+    "jdbc.not.suggest.column.names": "Could not suggest column names",
+    "jdbc.warning.delete.column.msg": "Are you sure you want to delete the column '{{columnName}}' ?",
+    "jdbc.preview.first.rows": "Preview of first 100 rows of table {{name}}",
+    "jdbc.not.show.preview": "Could not show preview",
+    "jdbc.table.definition": "The table definition produced no results",
+    "jdbc.preview.sql": "Preview SQL",
+    "jdbc.warning.invalid.query": "The data query must be a SELECT query",
+    "jdbc.invalid.query": "Invalid query",
+    "jdbc.warning.one.column.msg": "Please define at least one column",
+    "jdbc.invalid.columns": "Invalid columns",
+    "jdbc.warning.all.column.msg": "Please select SQL type for all columns, Invalid columns",
```

Comment:
These are two different messages. Invalid columns is header of the toastr error and the rest is body